guides/source/active_record_basics.md

```diff
@@ -21,27 +21,27 @@ What is Active Record?
 ----------------------
 
 Active Record is the M in [MVC][] - the model - which is the layer of the system
-responsible for representing business data and logic. Active Record facilitates
-the creation and use of business objects whose data requires persistent storage
-to a database. It is an implementation of the Active Record pattern which itself
-is a description of an Object Relational Mapping system.
+responsible for representing data and business logic. Active Record helps you
+create and use Ruby objects whose attributes require persistent storage
+to a database.
+
+Note that it's also possible to model data with Ruby objects that do *not* need to be backed by a database. [Active Model](active_model_basics.html) is used for that. So Active Record and Active Model together are the M in MVC.
```

Comment:
Minor suggestion:

```suggestion
Note that it's also possible to model data with Ruby objects that do *not* need to be backed by a database. [Active Model](active_model_basics.html) is commonly used for that in Rails, making Active Record and Active Model both part of the M in MVC, as well as your own plain Ruby objects.
```

And a thought: maybe we move this to an actual note / tip after the first two paragraphs?